docs/_includes/header.html

```diff
@@ -51,19 +51,43 @@
     </script>
     <script async src="https://www.google-analytics.com/analytics.js"></script>
     {% endif %}
-    <meta name="twitter:card" content="summary" />
+    <!-- HTML Meta Tags -->
+<title>Nighthawk</title>
+<meta name="description" content="The Nighthawk distribution for service mesh performance characterization. Integrating Meshery and Service Mesh Performance.">
+
+<!-- Facebook Meta Tags -->
+<meta property="og:url" content="https://getnighthawk.dev/">
+<meta property="og:type" content="website">
+<meta property="og:title" content="Nighthawk">
+<meta property="og:description" content="The Nighthawk distribution for service mesh performance characterization. Integrating Meshery and Service Mesh Performance.">
+<meta property="og:image" content="https://getnighthawk.dev/assets/images/logos/nighthawk/icon/nighthawk-logo.png">
+
+<!-- Twitter Meta Tags -->
+<meta name="twitter:card" content="summary_large_image">
+<meta property="twitter:domain" content="getnighthawk.dev">
+<meta property="twitter:url" content="https://getnighthawk.dev/">
+<meta name="twitter:title" content="Nighthawk">
+<meta name="twitter:description" content="The Nighthawk distribution for service mesh performance characterization. Integrating Meshery and Service Mesh Performance.">
+<meta name="twitter:image" content="https://getnighthawk.dev/assets/images/logos/nighthawk/icon/nighthawk-logo.png">
```

Comment:
@Vidyaa123 good stuff here. Will you kindly use the logo version that includes the dark text "Nighthawk" underneath the logo?

Reply:
Tried using the dark text Nighthawk stacked logo. The logos do not fit properly in both Facebook and Twitter previews. Attached are screenshots below. The image needs to be edited to fit properly.

 
<img width="1309" alt="Screenshot 2022-02-24 at 12 13 25" src="https://user-images.githubusercontent.com/15687491/155522499-ed4eccc7-dca3-4118-aec7-e59ec9d97b5a.png">
<img width="1212" alt="Screenshot 2022-02-24 at 12 14 02" src="https://user-images.githubusercontent.com/15687491/155522520-40957055-3991-49f9-b112-b49ace53d26a.png">

Reply:
Tried using the dark text Nighthawk stacked logo. The logos do not fit properly in both Facebook and Twitter previews. Attached are screenshots below. The image needs to be edited to fit properly.

<img width="1309" alt="Screenshot 2022-02-24 at 12 13 25" src="https://user-images.githubusercontent.com/15687491/155523480-285e1742-bbb3-4df9-b3d6-ced39e38fdf2.png">
<img width="1212" alt="Screenshot 2022-02-24 at 12 14 02" src="https://user-images.githubusercontent.com/15687491/155523495-e7712c0d-9e29-43d2-b68a-084349acc07e.png">

Reply:
I will look into this @Vidyaa123